ush/oceanice_nc2grib2.sh

```diff
@@ -0,0 +1,319 @@
+      -set_var VICE -set center 7 \
+      -set_date "${current_cycle}" -set_ftime "${aperiod} hour ave fcst" \
+      -set_scaling same same -set_grib_type c1 -grib_out "${outfile}"
+
+# Additional variables needed for GFSv17/GEFSv13 operational forecast
+# files, but GRIB2 parameters not available in NCEP (-set center 7)
+# tables in wgrib2 v2.0.8:
+
+#  -import_netcdf "${infile}" "hs_h" "0:1:${latlon_dims}" \
+#    -set_var ??? -set center 7 \
+#    -set_date "${current_cycle}" -set_ftime "${aperiod} hour ave fcst" \
+#    -set_scaling same same -set_grib_type c1 -grib_out "${outfile}" \
+#  -import_netcdf "${infile}" "frzmlt_h" "0:1:${latlon_dims}" \
+#    -set_var ??? -set center 7 \
+#    -set_date "${current_cycle}" -set_ftime "${aperiod} hour ave fcst" \
+#    -set_scaling same same -set_grib_type c1 -grib_out "${outfile}" \
+#  -import_netcdf "${infile}" "albsni_h" "0:1:${latlon_dims}" \
+#    -set_var ALBICE -set center 7 -rpn "100.0:/" \
+#    -set_date "${current_cycle}" -set_ftime "${aperiod} hour ave fcst" \
+#    -set_scaling same same -set_grib_type c1 -grib_out "${outfile}" \
+#  -import_netcdf "${infile}" "mlt_onset_h" "0:1:${latlon_dims}" \
+#    -set_var ??? -set center 7 \
+#    -set_date "${current_cycle}" -set_ftime "${aperiod} hour ave fcst" \
+#    -set_scaling same same -set_grib_type c1 -grib_out "${outfile}" \
+#  -import_netcdf "${infile}" "frz_onset_h" "0:1:${latlon_dims}" \
+#    -set_var ??? -set center 7 \
+#    -set_date "${current_cycle}" -set_ftime "${aperiod} hour ave fcst" \
+#    -set_scaling same same -set_grib_type c1 -grib_out "${outfile}"
+
+  rc=$?
+  # Check if the conversion was successful
+  if (( rc != 0 )); then
+   echo "FATAL ERROR: Failed to convert the ice rectilinear netCDF file to grib2 format"
+  fi
+  return "${rc}"
+
+}
+
+################################################################################
+function _ocean2D_nc2grib2 {
+# This function converts the ocean 2D rectilinear netCDF files to grib2 format
+
+  # Set the inputs
+  local grid=${1} # 0p25, 0p50, 1p00, 5p00
+  local latlon_dims=${2} # 0:721:0:1440, 0:361:0:720, 0:181:0:360, 0:36:0:72
+  local current_cycle=${3} # YYYYMMDDHH
+  local aperiod=${4} # 0-6
+  local infile=${5} # ocean.0p25.nc
+  local outfile=${6} # ocean_2D.0p25.grib2
+  local template=${7} # template.global.0p25.gb2
+
+  ${WGRIB2} "${template}" \
+  -import_netcdf "${infile}" "SSH" "0:1:${latlon_dims}" \
+    -set_var SSHG -set center 7 \
+    -set_date "${current_cycle}" -set_ftime "${aperiod} hour ave fcst" \
+    -set_scaling same same -set_grib_type c1 -grib_out "${outfile}" \
+  -import_netcdf "${infile}" "SST" "0:1:${latlon_dims}" \
+    -set_var WTMP -set center 7 -rpn "273.15:+" \
+    -set_date "${current_cycle}" -set_ftime "${aperiod} hour ave fcst" \
+    -set_scaling same same -set_grib_type c1 -grib_out "${outfile}" \
+  -import_netcdf "${infile}" "SSS" "0:1:${latlon_dims}" \
+    -set_var SALTY -set center 7 -rpn "1000.0:/" \
```

Comment:
Change `-set_var SALTY -set center 7 -rpn "1000.0:/ \`  to
`-set_var SALIN -set center 7 \`

Reply:
@GwenChen-NOAA @jiandewang 
I have updated the scripts to this suggestion and re-ran the test case. Please inspect the previously shared location for the ocean grib2 product.  For convenience, here is the location of the `f018` ocean 0.25 grib2 product file: `/scratch1/NCEPDEV/stmp2/Rahul.Mahajan/RUNTESTS/COMROOT/oi2/gfs.20160701/00/products/ocean/grib2/0p25/gfs.ocean.t00z.0p25.f018.grib2`

Reply:
> /scratch1/NCEPDEV/stmp2/Rahul.Mahajan/RUNTESTS/COMROOT/oi2/gfs.20160701/00/products/ocean/grib2/0p25/gfs.ocean.t00z.0p25.f018.grib2

salinity looks fine now

Reply:
@jiandewang Thank you for confirming the output looks correct now.
We will move to testing this PR now.

Reply:
Looks good to me too.
![ocean 0p25 SALIN z4481](https://github.com/NOAA-EMC/global-workflow/assets/95313292/13870cda-eac2-428f-86cd-6a5e60097663)
![ocean 0p25 SALIN z215](https://github.com/NOAA-EMC/global-workflow/assets/95313292/928ed9bf-a7ae-48cf-af8b-4031901e18b7)

Reply:
Thanks for confirming @GwenChen-NOAA